### PR TITLE
Create UI component for a releases' component

### PR DESCRIPTION
**Pull request by marians**
This provides a UI component for displaying the component of a release and the version changes, if any. That should simplify further work on https://github.com/giantswarm/giantswarm/issues/6400

## Preview

image

### CODE REVIEW

**Comment by marians** [comment]
Currently deployed on ginger

File: src/components/UI/release_component_label.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+const radius = '5px';
```
Comment (oponder) [inline src/components/UI/release_component_label.js]: could this come from the theme instead? https://github.com/giantswarm/happa/blob/master/src/components/UI/theme.js

Comment (marians) [reply src/components/UI/release_component_label.js]: As there is no 5px radius used by any other component, I think no. Possibly in the future.

File: src/components/cluster/detail/upgrade_cluster_modal.js [hunk 2]
```diff
@@ -46,12 +47,12 @@ class UpgradeClusterModal extends React.Component {
     }
 
     var components = {};
-    _.each(currentComponents, component => {
+    currentComponents.forEach(component => {
```
Comment (ograu) [inline src/components/cluster/detail/upgrade_cluster_modal.js]: @marians love it!

File: src/components/cluster/detail/upgrade_cluster_modal.js [hunk 3]
```diff
@@ -76,57 +77,33 @@ class UpgradeClusterModal extends React.Component {
...
-                    {component.name}
-                  </span>
-                  <span className='release-selector-modal--component--version'>
-                    <span className='lhs'>
-                      <span>{component.version}</span> <span>(removed)</span>
-                    </span>
-                  </span>
-                </div>
+                <ReleaseComponentLabel
+                  isRemoved={true}
```
Comment (ograu) [inline src/components/cluster/detail/upgrade_cluster_modal.js]: you can also do it as `isRemoved` there is no need for `={true}`

Comment (marians) [reply src/components/cluster/detail/upgrade_cluster_modal.js]: Very nice, thanks!

**Pull request merged**